### PR TITLE
Route WS Frontpages in simorgh

### LINKED ISSUE #1543: Route WS frontpages & add getInitalData function
**Is your feature request related to a problem? Please describe.**
We need to route the ws frontpages in simorgh

**Describe the solution you'd like**
Route the initial 3 services `Pidgin, Igbo, Yoruba`, ensuring they have fixture data and dont 404 when requested.

1) Setup an express route using the frontpage regex, which calls the code currently used [for articles](https://github.com/bbc/simorgh/blob/latest/src/server/index.jsx#L152-L167)

2) Create react router [route](https://github.com/bbc/simorgh/blob/latest/src/app/routes/index.js)

3) Create a `getFrontpageInitialData` function in the format of [the article one](https://github.com/bbc/simorgh/blob/latest/src/app/routes/getInitialData/article.js) requesting from the route of `${baseUrl}/${service}.json`and calling `fetchData({ url, preprocessorRules });` where `preprocessorRules` is an array of our 3 rules in the order of `filter item types`, `filter empty sections`, `squash top stories`

**End goal:**
When i hit `http://localhost:7080/igbo`, `http://localhost:7080/pidgin`, `http://localhost:7080/yoruba` I see the current frontpage with fixture data!

**Testing notes**
[Tester to complete]

Dev insight: CYPRESS TESTS! unit tests etc

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #1543

**Overall change:** Route the WS frontpages in Simorgh

**Code changes:**

- Routes ws frontpages in express and react router
- Creates get initial data function for simorgh

**BUNDLE SIZE CHANGES 18th JUNE:**
Before: 568kb
After: 577kb

Testing notes:
Regression existing routes like articles to ensure they havent broke, look for any glaring issues that are broken. Any small issues we can fix forward :)

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/routes/getInitialData/frontpage.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+];
+
+const getFrontpageInitialData = async ({ match }) => {
+  const { service } = match.params;
+
+  const baseUrl = onClient()
+    ? getBaseUrl(window.location.origin)
+    : process.env.SIMORGH_BASE_URL;
+
+  const url = `${baseUrl}/${service}.json`;
```
Comment (DenisHdz) [inline src/app/routes/getInitialData/frontpage.js:19]: Isn't it `${baseUrl}/${service}/frontpage/`; ?

Comment (dr3) [reply src/app/routes/getInitialData/frontpage.js:19]: Nope, the data route in simorgh, and soon in mozart is `http://thing.com/igbo.json` :)

**Review by chris-hinds — DISMISSED** [review]
LGTM, nice work and a nice little cleanup of the server 👍

**Review by DenisHdz — DISMISSED** [review]
Great job 👌🏻

**Comment by dr3** [comment]
Merging is blocked on cypress tests. Testing can still be done though

**Comment by david-boydell** [comment]
Hi Drew, when I request e.g. `http://localhost:7080/pidgin/` I get a 404, is this expected at the moment (e.g. is this not the final routing).

**Review by DenisHdz — DISMISSED** [review]

**Review by dr3 — COMMENTED** [review]
LGTM for the e2e's :) Obs I cant approve the rest :P

File: src/app/routes/getInitialData/frontpage.test.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+import filterEmptyGroupItems from '../../lib/utilities/preprocessor/rules/filterEmptyGroupItems';
+import applySquashTopstories from '../../lib/utilities/preprocessor/rules/topstories';
+
+const preprocessorRules = [
+  filterUnknownCpsTypes,
+  filterEmptyGroupItems,
+  applySquashTopstories,
+];
+
+process.env.SIMORGH_BASE_URL = 'https://www.SIMORGH_BASE_URL.com';
```
Comment (benjaminhobbs) [inline src/app/routes/getInitialData/frontpage.test.js]: this value is https://www.bbc.com in live, so this'd be https://www.https://www.bbc.com.com

**Review by jamesbhobbs — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]
LGTM 👍 🎉

**Comment by david-boydell** [comment]
All good, existing paths return 200, this is ready for merge. Nice.

**Pull request merged**